3rdparty/symcrypt_engine/CMakeLists.txt

```diff
@@ -131,6 +131,7 @@ enclave_compile_options(
   oesymcryptengine
   PRIVATE
   # SymCrypt OpenSSL engine flags
+  -std=c++17
```

Comment:
I dont know if we should rely on C++ here

Reply:
It was added there by mistake. I removed it